File: 06162023/38_151_Generating_Evens_Exercise/java/GeneratingEvensExercise.java
import java.util.ArrayList;

public class GeneratingEvensExercise {
    public static void main(String[] args) {
        int upperLimit = 50;
        ArrayList<Integer> evenNumbers = generateEvens(upperLimit);

        for (int number : evenNumbers) {
            System.out.println(number);
        }
    }

    public static ArrayList<Integer> generateEvens(int upperLimit) {
        ArrayList<Integer> numbers = new ArrayList<>();
    
        for (int i = 1; i < upperLimit; i++) {
            if (i % 2 == 0) {
                numbers.add(i);
            }
        }
    
        return numbers;
    }
}

File: 06162023/38_151_Generating_Evens_Exercise/python/generating_evens_exercise.py
#Define a function called generate_evens
#It should return a list of even numbers between 1 and 50(not including 50)
def generate_evens():
    answer = []

    for number in range(1, 50):
        if number % 2 == 0:
            answer.append(number)
    
    return answer